### PR TITLE
Update ofed driver version for redhat

### PR DESCRIPTION
**Pull request by contre**

### CODE REVIEW

File: lisa/features/infiniband.py [hunk 1]
```diff
@@ -152,10 +152,10 @@ def _get_mofed_version(self) -> str:
         default = "5.4-3.0.3.0"
         if self._is_legacy_device():
             return "4.9-5.1.0.0"
-        if (
-            isinstance(node.os, Ubuntu) and node.os.information.version >= "20.4.0"
-        ) or (isinstance(node.os, Redhat) and node.os.information.version >= "8.2.0"):
+        if isinstance(node.os, Ubuntu) and node.os.information.version >= "20.4.0":
             return "5.7-1.0.2.0"
+        if isinstance(node.os, Redhat) and node.os.information.version >= "8.2.0":
```
Comment (LiliDeng) [inline lisa/features/infiniband.py:157]: have you uploaded the ofed driver into our storage account?

Comment (contre) [reply lisa/features/infiniband.py:157]: Yes.

Comment (LiliDeng) [reply lisa/features/infiniband.py:157]: @contre I just double check, only MLNX_OFED_LINUX-5.8-1.1.2.1-rhel8.7-x86_64.tgz uploaded, that means, for other version, 8.6, 9.0, test case will fail for not find corresponding tar package

Comment (contre) [reply lisa/features/infiniband.py:157]: I will get that taken care of tomorrow.

Comment (LiliDeng) [reply lisa/features/infiniband.py:157]: please check #2453, I think it will fix the issue for rhel 8.7.

Comment (LiliDeng) [reply lisa/features/infiniband.py:157]: Hi @contre, #2453 has been merged, could you please check whether it fixed your issue or not? thanks.

**Comment by contre** [comment]
Should be fixed by the other PR

**Pull request closed without merging**